src/ashley/models.py

```diff
@@ -47,6 +47,10 @@ class AbstractUser(DjangoAbstractUser):
     REQUIRED_FIELDS: List[str] = []
 
     def get_public_username(self):
+        """Getter for the public username of the user."""
+        return self.public_username
+
+    def get_public_username_with_default(self):
```

Comment:
This method is used in the [sandbox settings](https://github.com/openfun/ashley/pull/132/files#diff-6149e0bb82135c43ba78910a0c8e83c4c6a6204a9900940574abe6b2ca53a23fR256).

The `default` template filter is only used in the search results template, because it does not receive a User model but a document from elasticsearch.

Reply:
oh. May I argue that it is not dry?

Reply:
Yes you may :smile: 
It's about the "Anonymous" string, or more than that ?

Reply:
Ok I see, not easy to do otherwise. Fair enough.